pet_wellness_app1/app/javascript/components/pages/Landing.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+
+class AddPet extends Component{
+    constructor(props){
+        super(props)
+        this.state = {
+            success: false
+        }
+    }
+
+    render(){
+        return(
+            <React.Fragment>
```

Comment:
Another thing you will see used often is the react fragment shorthand that does not require an import at all
```suggestion
            <>
```

Reply:
But you can ask me about that syntax later if you're interested